etl/steps/open_numbers.py

```diff
@@ -128,7 +128,13 @@ def load_and_combine(
                     f.write(line)
 
         f.flush()
-        df = pd.read_csv(f.name)
+
+        # raise warnings as exceptions since they would cause errors later on anyway
+        # if you see `pandas.errors.DtypeWarning: Columns (2) have mixed types.` make
+        # sure you set the proper type in `dtype`
+        with warnings.catch_warnings():
+            warnings.simplefilter("error")
+            df = pd.read_csv(f.name, dtype={"age_group_1year": "str"})
```

Comment:
This file processes a hundred datasets. I don't think patching a random column is a good approach.

A more general one:

- This situation only seems to happen with `load_and_combine()` because we're using low memory settings
- We can loop over the columns and use `pd.api.types.infer_dtype(col)`, checking for responses with the prefix "mixed-"
- For those columns
  - Try `pd.to_numeric(col)` first, catch the ValueError if it fails
  - Fallback to `col.astype('str')`

Reply:
Ok, I'll do that. I actually had that solution first, but I didn't want to unnecessarily complicate the code and make open numbers even slower.